tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectImportUtil.java

```diff
@@ -122,11 +119,10 @@ public static ProjectRecord createSDKPluginProject(
             IPath projectPath = null;
             IPath sdkPluginProjectFolder = liferaySDK.getLocation();
 
-            ILiferayRuntime liferayRuntime = ServerUtil.getLiferayRuntime( bridgedRuntime );
-            Map<String, String> appServerProperties = ServerUtil.getSDKRequiredProperties( liferayRuntime );
             // IDE-110 IDE-648
             String webappRootFolder = null;
             IProgressMonitor npm = new NullProgressMonitor();
+            String arguments = "\"" + displayName + "\"" + " " + "\"" + displayName + "\"";
```

Comment:
Arguments should not be escaped here but closing to where the create.bat script is invoked.